pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -138,6 +138,7 @@ cat >/etc/fluentbit/fluentbit.conf <<'EOF'
 	Name systemd
 	Tag journald
 	Systemd_Filter _COMM=aro
+	DB /var/lib/fluent/journaldb
```

Comment:
We should probably also add that to the gateway vmss

Reply:
done